internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -110,12 +110,13 @@ func ParseConnection(c Connection) (sdktypes.Connection, error) {
 
 type Var struct {
 	// varID is scopeID. just mapped directly for reusing the join code
-	VarID      sdktypes.UUID `gorm:"primaryKey;index;type:uuid;not null"`
-	ScopeID    sdktypes.UUID `gorm:"-"`
-	Name       string        `gorm:"primaryKey;index;not null"`
-	Value      string
-	IsSecret   bool
-	IsOptional bool
+	VarID       sdktypes.UUID `gorm:"primaryKey;index;type:uuid;not null"`
+	ScopeID     sdktypes.UUID `gorm:"-"`
+	Name        string        `gorm:"primaryKey;index;not null"`
+	Value       string
+	Description string
+	IsSecret    bool
+	IsOptional  bool
```

Comment:
Throughout this PR you define/use these parameters in a different order every time.